jemoji.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'jekyll', '>= 3.0'
   s.add_dependency 'html-pipeline', '~> 2.2'
   s.add_dependency 'activesupport', '~> 4.0'
-  s.add_dependency 'gemoji', '~> 2.0'
+  s.add_dependency 'gemoji', '~> 3.0'
 
```

Comment:
I was only curious. If there is no reason for users to be running 2.0 then this is fine :+1: